core/kazoo/src/kz_maps.erl

```diff
@@ -0,0 +1,70 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2017, 2600Hz INC
+%%% @doc
+%%% map utilities
+%%% @end
+%%% @contributors
+%%%-------------------------------------------------------------------
+-module(kz_maps).
+
+%% ====================================================================
+%% API functions
+%% ====================================================================
+-export([merge/2]).
+-export([get/2, get/3]).
+-export([keys_to_atoms/1, keys_to_atoms/2]).
+
+-spec merge(map(), map()) -> map().
+merge(Map1, Map1) -> Map1;
```

Comment:
Can you add unit tests so we can see the intended usages/results? I think there may be more efficient ways to do this merge but I wouldn't want to break functionality, so tests would be great to make sure there isn't a regression.